.github/workflows/dockerfile-ci.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow to help you get started with Actions
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [dockerfile-ci]
```

Comment:
This should be the `develop` branch. Also the file should be renamed in the `workflows/` directory to be `develop.yml` instead of `dockerfile-ci.yml` and probably a separate file for one for `main` as well.